source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -195,12 +210,12 @@
         this.period = data.Period;
         this.currentPosition= data.CurrentPosition || '';
         this.seniority = data.Seniority || '';
-        this.evaluators = data.Evaluators;
+        this.evaluators(data.Evaluators);
         this.evaluatorsString = ko.computed(function () {
-            return this.evaluators.toString().replace(/,/g, ', ');
+            return this.evaluators().toString().replace(/,/g, ', ');
```

Comment:
I never used `Array.toString()`, I did not know that it returns a list of values comma separated. It works in this way in all browsers?